floor_generator/lib/writer/transaction_method_writer.dart

```diff
@@ -14,7 +14,7 @@ class TransactionMethodWriter implements Writer {
     return Method((builder) => builder
       ..annotations.add(overrideAnnotationExpression)
       ..returns = refer(method.returnType.getDisplayString(
-        withNullability: false,
+        withNullability: true,
```

Comment:
❓ Do we also get support for the case described in https://github.com/vitusortner/floor/issues/557 where we don't return a nullable `Future` but a `Future` wrapping a nullable type (e.g `Future<Task?>`)? If so, could you add another test to verify the behavior?

Reply:
I think we had that support already, but I'll add a test.